code/modules/library/lib_items.dm

```diff
@@ -61,7 +61,7 @@
 				state = 0
 
 		if(2)
-			if(istype(I, /obj/item/weapon/book) || istype(I, /obj/item/weapon/spellbook))
+			if(istype(I, /obj/item/weapon/book) || istype(I, /obj/item/weapon/spellbook) || istype(I, /obj/item/weapon/storage/book))

```

Comment:
Can you add the bible to this as well? Oh, and do the same to the book bag pickup list.